jiant/tasks/lib/templates/squad_style/core.py

```diff
@@ -117,31 +117,32 @@ def to_feature_list(
         spans = []
 
         truncated_query = tokenizer.encode(
-            self.question_text, add_special_tokens=False, max_length=max_query_length
+            self.question_text,
+            add_special_tokens=False,
+            truncation=True,
+            max_length=max_query_length,
         )
         sequence_added_tokens = (
             tokenizer.max_len - tokenizer.max_len_single_sentence + 1
-            if "roberta" in str(type(tokenizer))
+            if "roberta" in str(type(tokenizer)) or "camembert" in str(type(tokenizer))
             else tokenizer.max_len - tokenizer.max_len_single_sentence
         )
         sequence_pair_added_tokens = tokenizer.max_len - tokenizer.max_len_sentences_pair
 
         span_doc_tokens = all_doc_tokens
         while len(spans) * doc_stride < len(all_doc_tokens):
 
-            encoded_dict = tokenizer.encode_plus(
+            encoded_dict = tokenizer.encode_plus(  # TODO(thom) update this logic
```

Comment:
What do you mean by this?

Reply:
Inherited comment from the HF implementation. Since a lot of the code is take unmodified, I'm leaning towards leaving this in.